docs/core/tools/dotnet.md

```diff
@@ -5,212 +5,157 @@ ms.date: 02/13/2020
-
-`-d|--diagnostics`
-
-Enables diagnostic output.
-
-`--fx-version <VERSION>`
-
-Version of the .NET Core runtime to use to run the application.
-
-`-h|--help`
-
-Prints out documentation for a given command, such as `dotnet build --help`. `dotnet --help` prints a list of available commands.
-
-`--info`
-
-Prints out detailed information about a .NET Core installation and the machine environment, such as the current operating system, and commit SHA of the .NET Core version.
-
-`--list-runtimes`
-
-Displays the installed .NET Core runtimes.
-
-`--list-sdks`
-
-Displays the installed .NET Core SDKs.
-
-`--roll-forward-on-no-candidate-fx <N>`
-
-Defines behavior when the required shared framework is not available. `N` can be:
-
-- `0` - Disable even minor version roll forward.
-- `1` - Roll forward on minor version, but not on major version. This is the default behavior.
-- `2` - Roll forward on minor and major versions.
-
- For more information, see [Roll forward](../whats-new/dotnet-core-2-1.md#roll-forward).
-
-`--runtimeconfig`
+To run an application (.NET Core SDK 3.x):
 
-Path to a *runtimeconfig.json* file.
-
-A *runtimeconfig.json* file is a configuration file containing run-time settings. For more information, see [.NET Core run-time configuration settings](../run-time-config/index.md#runtimeconfigjson).
+```dotnetcli
+dotnet [--additionalprobingpath] [--deps-file] [--additional-deps]
+    [--fx-version]  [--roll-forward]
+    [path-to-application] [arguments]
+```
 
-`-v|--verbosity <LEVEL>`
+## Description
 
-Sets the verbosity level of the command. Allowed values are `q[uiet]`, `m[inimal]`, `n[ormal]`, `d[etailed]`, and `diag[nostic]`. Not supported in every command; see specific command page to determine if this option is available.
+The `dotnet` command has two functions:
 
-`--version`
+- It manages .NET Core source code and binaries.
 
-Prints out the version of the .NET Core SDK in use.
+  You specify commands that perform specific tasks. For example, [`dotnet build`](dotnet-build.md) builds a project. Each command defines its own options and arguments. All commands support the `--help` option for printing out brief documentation about how to use the command.
 
-# [.NET Core 2.0](#tab/netcore20)
+- It runs .NET Core applications.
 
-`--additional-deps <PATH>`
+  You specify the path to an application `.dll` file to run the application. For example, `dotnet myapp.dll` runs the `myapp` application. See [.NET Core application deployment](../deploying/index.md) to learn about deployment options.
 
-Path to an additional *.deps.json* file.
+## Options
 
-`--additionalprobingpath <PATH>`
+Different options are available for `dotnet` by itself, for running a command, and for running an application.
 
-Path containing probing policy and assemblies to probe.
+### SDK options for dotnet by itself
```

Comment:
This is not SDK specific - these commands work always, even if SDK is not installed.